src/components/EventCalender/Monthly/EventCalender.tsx

```
@@ -351,6 +365,9 @@ const Calendar: React.FC<
   };
 
   const renderDays = (): JSX.Element[] => {
+    const { t: tCalendar } = useTranslation('translation', {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)